infra/gcp/clusters/projects/artifact-management-sandbox/main.tf

```diff
@@ -0,0 +1,57 @@
+
+/**
+ * Copyright 2021 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ locals {
+     project_id = "artifact-management-sandbox"
+ }
+
+data "google_organization" "org" {
+  domain = "kubernetes.io"
+}
+
+
+resource "google_project" "project" {
+    name = local.project_id
+    project_id = local.project_id
+    org_id = data.google_organization.org.org_id
+}
+
+
+resource "google_project_service" "project" {
+  project = google_project.project.id
+
+  for_each = toset([
+    "cloudbuild.googleapis.com",
+    "container.googleapis.com",
+    "containeranalysis.googleapis.com",
+    "containerregistry.googleapis.com",
+    "oslogin.googleapis.com",
+    "storage-component.googleapis.com"
```

Comment:
Supposed to be the same services as `k8s-artifacts-prod` : https://github.com/kubernetes/k8s.io/issues/1758#issuecomment-822685501. 
I would like the rest of the services added by hh's team.

Reply:
👍

Reply:
Will update this once we merge / we explore current access.